components/feature/accounts/src/main/java/mozilla/components/feature/accounts/FxaWebChannelFeature.kt

```diff
@@ -40,14 +48,16 @@ import java.util.WeakHashMap
  * @property engine a reference to application's browser engine.
  * @property sessionManager a reference to application's [SessionManager].
  * @property accountManager a reference to application's [FxaAccountManager].
+ * @property fxaCapabilities a set of [FxaCapability] that client supports.
  */
 @Suppress("TooManyFunctions")
 class FxaWebChannelFeature(
     private val context: Context,
     private val customTabSessionId: String?,
     private val engine: Engine,
     private val sessionManager: SessionManager,
-    private val accountManager: FxaAccountManager
+    private val accountManager: FxaAccountManager,
+    private val fxaCapabilities: Set<FxaCapability>? = emptySet()
```

Comment:
Do we need this to be null-able?

Reply:
I did that to simply the consumer API. E.g. instead of passing in `emptySet()`, you can just omit it.

Reply:
Yeah, but as we already have `emptySet()` as default value, the consumer doesn't have to provide a value for this property. Null aren't that nice it's better if we can keep them out :)

Reply:
Ah! Duh... Friday brain :) You're right :) I'll fix this.